### ISSUE TITLE
UI: Horizontal scrollbar still appearing when browser is maximized.

### ISSUE THREAD

**Issue by PQ-ops-PQ** [asgardex/asgard-exchange#189]
in my opinion, the scrollbar shouldn't appear when you have nothing to scroll.
Tested in browser: chrome, edge
![image](https://user-images.githubusercontent.com/81558774/112814694-21c3f180-90b2-11eb-89fe-0f278c931805.png)

**Comment by stuartkuentzel** [comment]
Hey @PQ-ops-PQ thanks for pointing this out. I tested this out in Chrome (on a Mac) and couldn't replicate the issue. Was there anything noticeably overflowing horizontally? I'm thinking it might be a Windows thing, as the scrollbars are different than Mac. I pushed a potential fix https://github.com/asgardex/asgard-exchange/pull/198.

Let me know if that clears the horizontal scrollbar for you.
